app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardsController < ApplicationController
+  def show
+    @environment = Environment.find(params[:id])
+    @data = Project.alphabetical.each_with_object({}) do |project, hash|
+      hash[project] = project.last_release_by_deploy_group
```

Comment:
we could avoid some sql work by passing in the groups we are interested in `last_by_group(@environment.deploy_group_ids)` and then skip the `select!`